src/views/AccountDetail.vue

```diff
@@ -39,7 +39,7 @@
         </template>
       </Card>
 
-      <Card v-if="1||showNamespaces" class="card-f card-adaptive" :loading="loading">
```

Comment:
i think `showNamespaces ` is valild because it existing in `computed` , only `1||` have to remove.